recipes-devtools/cppmicroservices/cppmicroservices_4.0.bb

```diff
@@ -0,0 +1,63 @@
+inherit cmake
+
+SUMMARY = "C++ Micro Services"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=3b83ef96387f14655fc854ddc3c6bd57"
+
+# Packages required to build
+DEPENDS_append_class-target += "cppmicroservices-native"
+
+# Parallel make does not work with some recipes
+PARALLEL_MAKE = ""
+PARALLEL_MAKEINST = ""
+
+### source ###
+
+# Source repo
+SRC_URI = "gitsm://github.com/intel/linux-sgx.git"
+SRCREV = "a21f2d9dd77e7672b00f99f9b61bc81cc44e5954"
+
+# Patches
+SRC_URI_append_class-target += "file://0001-host-bin.patch"
+
+# Source directory
+S = "${WORKDIR}/git/external/CppMicroServices"
+
+INHIBIT_PACKAGE_STRIP = "1"
+INHIBIT_SYSROOT_STRIP = "1"
+INHIBIT_PACKAGE_DEBUG_SPLIT  = "1"
+
+### configure ###
+
+HOST_BIN = "${datadir}/cppmicroservices4/host_bin"
+
+EXTRA_OECMAKE += "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+EXTRA_OECMAKE_append_class-target += "-DCMAKE_HOST_BINARY_DIR:PATH=${RECIPE_SYSROOT_NATIVE}${HOST_BIN}"
+
+### install ###
+
+do_install_append () {
+    rm -rfv ${D}${libdir}/.debug
+}
+
+do_install_append_class-native () {
+    install -d "${D}${datadir}/cppmicroservices4/host_bin"
+    install -m 0644 "${B}/ImportExecutables.cmake" "${D}${HOST_BIN}"
+    install -m 0755 "${B}/bin/usResourceCompiler4" "${D}${HOST_BIN}"
+}
+
+do_install_append_class-target () {
+    if [ "${libdir}" != "/usr/lib" ]; then
```

Comment:
/usr/lib are we referring host or target in yocto ? wondering if we need some yocto bindir or libdir prefixes ?

Reply:
/usr/lib is install target of make install of cppmicroservices. It is not libdir or bindir from yocto.